src/function/probability/productrange.js

```diff
@@ -0,0 +1,14 @@
+
+function product (i, n) {
+  let half
+  if (n < i) {
+    return 1
+  }
+  if (n === i) {
+    return n
+  }
+  half = ((n + i) / 2) | 0
```

Comment:
Does `| 0` round towards zero here? Maybe `Math.trunc` would be clearer if so?

Reply:
Hi, math.trunc is slower compared to operators that convert to integers. I decided to use the >> operator instead since I am gonna dividing by 2 and truncate anyways. I did leave a comment in product.js to clarify what it does.

Reply:
:+1: